src/invoice2data/output/to_csv.py

```diff
@@ -48,7 +50,7 @@ def write_to_file(data, path):
             csv_items = []
             for k, v in line.items():
                 # first_row.append(k)
-                if k == 'date':
-                    v = v.strftime('%d/%m/%Y')
+                if k.startswith('date') or k.endswith('date'):
```

Comment:
`if 'date' in k:`

Reply:
Are you suggering to ignore `startswith()` and `endswith()` and only apply format for `date` key ?